upwork/desktop_app.py

```diff
@@ -1,6 +1,12 @@
+from __future__ import print_function
+
 import upwork
 from pprint import pprint
 
+try:
```

Comment:
`six` package provides compatibility layer between Py2 and Py3. For example for raw_input you can use
```
import six
six.moves.input('...')
```
http://six.readthedocs.io/index.html?highlight=raw_input